test/fixtures/locations.yml

```diff
@@ -258,7 +258,10 @@ perkatkun:
   north: 71.154802
   west: -179.809370
   south: 71.104900
-  east: -179.613214
+  # NOTE: jdc 2024-07-04
+  # Next line causes failures. Due to AR/Ruby truncation/rounding??
+  # east: -179.613214
+  east: -179.613
```

Comment:
@mo-nathan: Can you shed any light on this? 
When I had `east: -179.613214`:
1. If I put a breakpoint anywhere in the tests and do an AR query for this fixture's `east`, then the result is rounded to 6 digits plus the minus sign. Ex:
```ruby 
=> 631|     debugger
   632|     # loc doesn't straddle 180
   633|     #   potential br (bounding rectangle) entirely to "left" of loc
   634|     do_contains_box(loc: albion, external_loc: perkatkun,
   635|                     regions: [california, earth])
=>#0    LocationTest#test_scope_contains_box at ~/mushroom-observer/test/models/location_test.rb:631
  #1    block in run (3 levels) at ~/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/minitest-5.23.1/lib/minitest/test.rb:95
  # and 25 frames (use `bt' command for all frames)
(ruby) perkatkun.east
-179.613
```
I don't understand why, or what implications that might have outside of this PR.

2. Tests fail, and I don't know how to fix them. For instance, increasing `ROUND_ERROR` didn't work.
My change to the fixture makes the tests pass, but it dodges the problem rather than solving it.

Reply:
What should I do if `Location.contains` called with missing/bad arguments:
1. Currently I think it throws an error? This alerts us to the problem.
2. Return Location.none
3. Other?

Reply:
I vote error.